taxi_service/urls.py

```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
-
+from django.urls import path, include
+from taxi.views import index
 urlpatterns = [
```

Comment:
one blank line between different groups of imports. 2 blank lines between imports and code 
Follow pep8, please